components/gitpod-cli/cmd/tasks-attach.go

```diff
@@ -116,6 +115,7 @@ var attachTaskCmd = &cobra.Command{
 			ForceResize: forceResize,
 			Interactive: interactive,
 		})
+		return nil
```

Comment:
we should return an error from attach

Reply:
if it is exti code from underlying terminal returned by supervisor event then it should be a user error, if it is related to communications with supervisor to support attach then it should be a system error

Reply:
Sounds good